src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs

```diff
@@ -27,19 +27,12 @@ namespace Tizen.NUI.BaseComponents
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class ImageViewStyle : ViewStyle
     {
-        private bool? preMultipliedAlpha;
-        private RelativeVector4 pixelArea;
-        private bool? borderOnly;
-        private bool? synchronosLoading;
-        private bool? orientationCorrection;
-
-        static ImageViewStyle() { }
-
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty ResourceUrlSelectorProperty = BindableProperty.Create("ResourceUrlSelector", typeof(Selector<string>), typeof(ImageViewStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var imageViewStyle = (ImageViewStyle)bindable;
+            if (null == imageViewStyle.resourceUrlSelector) imageViewStyle.resourceUrlSelector = new Selector<string>();
             imageViewStyle.resourceUrlSelector.Clone((Selector<string>)newValue);
```

Comment:
When the `newValue` is null, next line will cause crash. Is it OK not to test new value's nullity here?

Reply:
Yes. I'll add null check.

Reply:
Yes. I'll add null value check